main.go

```diff
@@ -134,7 +134,7 @@ func (opts *options) Parse() {
 	// Leader Election options flags
 	flag.BoolVar(&opts.enableLeaderElection, "enable-leader-election", true,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&opts.leaderElectionResourceLock, "leader-election-resource", "configmaps", "determines which resource lock to use for leader election. option:[configmapsleases|endpointsleases|configmaps]")
+	flag.StringVar(&opts.leaderElectionResourceLock, "leader-election-resource", "configmapsleases", "determines which resource lock to use for leader election. option:[configmapsleases|endpointsleases|leases]")
```

Comment:
nit - there is an actual constant for this https://pkg.go.dev/k8s.io/client-go@v0.23.5/tools/leaderelection/resourcelock#pkg-constants